src/main/java/com/google/sps/data/ComputeAPIClientInterface.java

```diff
@@ -11,31 +11,37 @@
    * are executed.
    * @param {String} operation - the name of the method that should be called.
    * @param {String} instanceName - the name of the instance being operated on.
-   * @param {Optional<String>} password - the password needed to access this 
-   *    instance. Not all operations requires a password.
+   * @param {Optional<String>} sessionId - the session id associated with
+   *     this instance. 
    */
-  void helper(String operation, String instanceName, 
-    Optional<String> password);
+  void computeManager(String operation, String instanceName, 
+    Optional<String> sessionId);
```

Comment:
Dont call this session id. The ComputeAPIClient shouldn't know *anything* about virtual movie night. Rename this to vncServerPassword. Please create descriptive variable and function names. 
Also why is it optional? Does it make sense to create a VM without providing a password? Probably not, just remove the optional.